interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
+        // Determine what text to show as a tooltip for this search_item.
+        if (SampleBin.hasItem(search_item)) {
+          return 'already added to analysis';
```

Comment:
I strongly object to `ADD` in all caps, but if you wanted initial caps to be our style on button labels I'd be okay with that. @jaclyn-taroni, can you explain why you'd want the add button to YELL?

I do not have a strong opinion about tool tips, so I'll make those sentence case.

- [ ] change tool tips to sentence case